sdk/src/lib.rs

```diff
@@ -10,6 +10,7 @@ pub use signer::signers;
 pub use solana_program::*;
 
 pub mod account;
+pub mod account_context;
```

Comment:
Needs different name, could be in their own files, or maybe `transaction_context.rs` which is a super group of the rest